src/Session/Session.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Pantheon\Terminus\Session;
+
+use Terminus\Caches\FileCache;
+use Terminus\Models\User;
+
+class Session {
+  /**
+   * @var FileCache
+   */
+  protected $cache;
+
+  /**
+   * @var object
+   */
+  protected $data;
+
+  /**
+   * Instantiates object, sets session data
+   */
+  public function __construct() {
+    $this->cache = new FileCache();
```

Comment:
Yes. I was thinking that. I decided to go one step at a time though until I was sure that I had the injection stuff right in the first place.